nemo/backends/pytorch/common/losses.py

```diff
@@ -34,24 +34,8 @@ class SequenceLoss(LossNM):
     @property
     def input_ports(self):
         """Returns definitions of module input ports.
```

Comment:
What was wrong with the documentation of that function?

Have you added some method to automatically extract description of input/output ports so they will appear in documentation?

Reply:
"Have you added some method to automatically extract description of input/output ports so they will appear in documentation?" 
- No. This isn't trivial so I plan to address it as a part of documentation PR.
I also need to implement print(neural_type_object) so that it gives meaningfull message.
Would be cool if I could add port definitions in the docs automatically - not sure how yet but will try.